src/nvim/ex_cmds2.c

```diff
@@ -3084,6 +3042,64 @@ int do_source(char_u *fname, int check_other, int is_vimrc)
 }
 
 
+// Check if fname was sourced before to finds its SID.
+// If it's new, generate a new SID.
+// @param[in] fname file path of script
+// @param[out] ret_sctx sctx of this script
```

Comment:
docstrings need 3 `///`
```suggestion
/// Check if fname was sourced before to finds its SID.
/// If it's new, generate a new SID.
/// @param[in] fname file path of script
/// @param[out] ret_sctx sctx of this script
```